compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -534,3 +539,11 @@ internal class SyntheticMouseEvent(
     x: Int,
     y: Int
 ) : MouseEvent(source, id, `when`, modifiers, x, y, 0, false)
+
+
+internal val java.util.Locale.layoutDirection: LayoutDirection
```

Comment:
We also have internal API inside `LayoutConfiguration.desktop.kt`, which is used in some parts. Let's move this function there, and implement `GlobalLayoutDirection` and `Component.layoutDirection`. To implement GlobalLayoutDirection, we probably have to create a fake component:
```
private val component = object : Component() {}
```

By the way, worth to mention that this PR partially fixes https://github.com/JetBrains/compose-jb/issues/872

Reply:
It seems that fake Component is not necessary, since we can get the layout direction from `Locale.getDefault()`. Added it in the last commit.